### DISCUSSION TITLE
Trame Teleinfo

### DISCUSSION THREAD

**Discussion by hallard** [hallard/tinfo_replay#1] [General]
## 👋 Bienvenue!
  Vous pouvez utiliser ce fil pour uploader des trames téléinfo récupérée avec le logiciel de ce repo 💪.

**Comment by Vincent835** [comment]
Voici 2 enregistrements.
[teleinfo_base_triphasé_standard.txt](https://github.com/hallard/tinfo_replay/files/6320299/teleinfo_base_triphase_standard.txt)
[teleinfo_base_triphasé_historique.txt](https://github.com/hallard/tinfo_replay/files/6320300/teleinfo_base_triphase_historique.txt)
